How do i install it?

Hi i am new to zig and i would like to use this library for my project. But I don't see any install instructions on README so idk how can i use it.

Any insight on using the Zig Package Manager?

```bash
$ zig version
0.12.0-dev.3212+40e64245f
```

`build.zig.zon`:

```zig
.{
    .name = "package-name-here",
    .version = "0.0.0",
    .dependencies = .{
        .clap = .{
            .url = "https://github.com/Hejsil/zig-clap/archive/refs/heads/master.tar.gz",
            .hash = "12203896de6eedec14712f4f1eaac8b646939cfed213c56accf231a0abb05f9dbb77",
         },
    },
    .paths = .{
        "",
    },
}
```

`build.zig`:

```zig
const clap = b.dependency("clap", .{});
exe.root_module.addImport("clap", clap.module("clap"));
exe.linkLibrary(clap.artifact("clap"));
```

And when I try to `zig build`:

```
$ zig build
thread 39008 panic: unable to find artifact 'clap'
C:\Users\user\.zvm\master\lib\std\debug.zig:434:22: 0x9338f4 in panicExtra__anon_18765 (build.exe.obj)
    std.builtin.panic(msg, trace, ret_addr);
                     ^
C:\Users\user\.zvm\master\lib\std\debug.zig:409:15: 0x908bac in panic__anon_17759 (build.exe.obj)
    panicExtra(null, null, format, args);
              ^
C:\Users\user\.zvm\master\lib\std\Build.zig:1790:18: 0x8dab74 in artifact (build.exe.obj)
            panic("unable to find artifact '{s}'", .{name});
                 ^
C:\Users\user\workspace\build.zig:28:34: 0x88f656 in build (build.exe.obj)
    exe.linkLibrary(clap.artifact("clap"));
                                 ^
C:\Users\user\.zvm\master\lib\std\Build.zig:1992:33: 0x86dfa5 in runBuild__anon_8716 (build.exe.obj)
        .Void => build_zig.build(b),
                                ^
C:\Users\user\.zvm\master\lib\build_runner.zig:310:29: 0x868f5a in main (build.exe.obj)
        try builder.runBuild(root);
                            ^
C:\Users\user\.zvm\master\lib\std\start.zig:350:53: 0x86ffdc in WinStartup (build.exe.obj)
    std.os.windows.ntdll.RtlExitUserProcess(callMain());
                                                    ^
???:?:?: 0x7ff80d99257c in ??? (KERNEL32.DLL)
???:?:?: 0x7ff80f0eaa57 in ??? (ntdll.dll)
error: unable to read results of configure phase from 'C:\Users\user\workspace\zig-cache\tmp\ba9a75b7430cf0ff': FileNotFound
```

If I remove `exe.linkLibrary(clap.artifact("clap"));`, the build works, but I can't run the application:

```
$ zig run src/main.zig -- -h
src\main.zig:2:22: error: no module named 'clap' available within module main
const clap = @import("clap");
                     ^~~~~~
referenced by:
    main: src\main.zig:13:29
    callMain: C:\Users\user\.zvm\master\lib\std\start.zig:511:32
    remaining reference traces hidden; use '-freference-trace' to see all reference traces
```

I've tried using Zig `0.11.0` with tag `0.7.0` as well (and converted the `build.zig` file to match) with basically the same results:

```
$ zig run src/main.zig -- -h
src\main.zig:2:22: error: no package named 'clap' available within package 'root'
const clap = @import("clap");
                     ^~~~~~
referenced by:
    main: src\main.zig:13:29
    callMain: C:\Users\user\.zvm\bin\lib\std\start.zig:574:32
    remaining reference traces hidden; use '-freference-trace' to see all reference traces
```

--- a/README.md
+++ b/README.md
@@ -10,6 +10,21 @@ version of zig-clap that targets a specific Zig release, have a look
 at the releases. Each release specifies the Zig version it compiles with
 in the release notes.
 
+## Installation
+
+First, run the following:
+
+```
+zig fetch --save git+https://github.com/Hejsil/zig-clap
+```
+
+Then add the following to `build.zig`:
+
+```zig
+const clap = b.dependency("clap", .{});
+exe.root_module.addImport("clap", clap.module("clap"));
+```
+
 ## Features
 
 * Short arguments `-a`

--- a/example/README.md.template
+++ b/example/README.md.template
@@ -10,6 +10,21 @@ version of zig-clap that targets a specific Zig release, have a look
 at the releases. Each release specifies the Zig version it compiles with
 in the release notes.
 
+## Installation
+
+First, run the following:
+
+```
+zig fetch --save git+https://github.com/Hejsil/zig-clap
+```
+
+Then add the following to `build.zig`:
+
+```zig
+const clap = b.dependency("clap", .{{}});
+exe.root_module.addImport("clap", clap.module("clap"));
+```
+
 ## Features
 
 * Short arguments `-a`

